src/components/AeList.vue

```diff
@@ -0,0 +1,12 @@
+<template>
```

Comment:
Is this component necessary? [SettingsNetwork component](https://github.com/aeternity/aepp-identity/blob/6d94730/src/pages/SettingsNetwork.vue) has a similar structure, and it looks okay without it.